build-index/tagclient/client.go

```diff
@@ -70,6 +71,14 @@ func NewSingleClient(addr string, config *tls.Config) Client {
 	return &singleClient{addr, config}
 }
 
+func (c *singleClient) CheckReadiness() error {
+	_, err := httputil.Get(
+		fmt.Sprintf("http://%s/readiness", c.addr),
+		httputil.SendTimeout(5*time.Second),
```

Comment:
Same as #374 can we make this configurable.

Reply:
I answered there why I believe we shouldn't do this: https://github.com/uber/kraken/pull/374#discussion_r1842862462